t/44-scripts.t

```diff
@@ -38,9 +38,9 @@ for my $script (sort keys %types) {
     my $rc  = $?;
     is($rc, 0, "Calling '$script --help' returns exit code 0")
       or diag "Output: $out";
-    $out = qx{$Bin/../script/$script --invalid-option 2>&1};
+    $out = qx{$Bin/../script/$script invalid-command --invalid-flag 2>&1};
```

Comment:
hm, this is then not checking anymore that *only* an invalid flag will also return a non-zero code. Any idea how to cover both without excessively increase the runtime of t/44-scripts.t?

Reply:
I've already mentioned in the commit message that the tests of Mojolicious itself cover this. I don't see a reason to replicate them here. For non-Mojolicious scripts this change shouldn't make a difference.

Reply:
right, good enough.